jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -100,6 +94,16 @@ def ConvertAndCompare(self, func_jax: Callable, *args,
         modes; when there is no exception the result should be the same
         as in JAX.
     """
+    original_impl = jax2tf.jax2tf.TensorFlowTrace.get_primitive_impl
+
+    def patch_get_primitive_impl():
+      wrapper = jcs.collect_limitations
+      jax2tf.jax2tf.TensorFlowTrace.get_primitive_impl = ( # type: ignore
+        lambda s, p: wrapper(p, original_impl(s, p)))
+
+    def restore_get_primitive_impl():
```

Comment:
There is a useful pattern that you can use here. Check out https://docs.python.org/2/library/unittest.html#unittest.TestCase.addCleanup